### DISCUSSION TITLE
Discussion for IdentityModel upgrade

### DISCUSSION THREAD

**Discussion by HaoK** [dotnet/aspnetcore#23728] [General]
Discussion issue for https://github.com/aspnet/Announcements/issues/428

**Comment by NinoFloris** [comment]
Any chance a bump to 6.7.0 (released 10 days ago) can be considered, it's small but seems to contain important fixes?

Also according to release notes, breaking API changes did happen in 6.5.0.

https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/releases/tag/6.5.0
>Breaking changes from 5.6.0:
The OpenIdConnectMessage(JObject json) constructor was removed and has been replaced with OpenIdConnectMessage(object json).
    TokenValidationResult has been moved from Microsoft.IdentityModel.JsonWebTokens to Microsoft.IdentityModel.Tokens.
    The JsonWebKeySet(string json, JsonSerializerSettings jsonSerializerSettings) constructor has been removed.

**Reply by Tratcher** [reply]
> Any chance a bump to 6.7.0 (released 10 days ago) can be considered, it's small but seems to contain important fixes?

We're considering it. It should be safe for you to update that locally if needed.

> Also according to release notes, breaking API changes did happen in 6.5.0.

Yeah, we checked and didn't find any API breaks that directly affected Asp.Net scenarios.
